### DISCUSSION TITLE
How to register as a team

### DISCUSSION THREAD

**Discussion by AshleyZhao** [microsoft/AI-Chat-App-Hack#19] [General]
Hi, while registering as a team, there's only the option to put down 1 person's name. Would other people need to register as well?

**Comment by pamelafox** [comment]
Each person should register individually. When submitting your project at the end, you'll be able to specify your team members (if any).

**Comment by pallavigadepalli** [comment]
ah ok because we register as a team and we want to submit together at the end.

**Comment by pallavigadepalli** [comment]
We are registered as a team and I am asking my team now to register individually. Can you let me know how to merge later? we are geographically distributed. I dont want to write code but I would love to try to do nocode and no code approaches. 
my team likes to build code.

**Reply by multispark** [reply]
When you submit your project, just make sure to tag your teammates on the submission!
